### PR TITLE
FIX Permissions on DNDataArchives weren't creating unique cache keys.

### PR DESCRIPTION
**Pull request by micmania1**
This fixes permission checks for listing snapshots.

### CODE REVIEW

**Comment by ss23** [comment]
Looks A+ apart from that random indentation :)

File: code/control/DNRoot.php [hunk 1]
```diff
@@ -1336,7 +1336,7 @@ protected function getCurrentEnvironment($project = null) {
 	 */
 	public function CanViewArchives(Member $member = null) {
 		if(!$member) $member = Member::currentUser();
-
+
```
Comment (halkyon) [inline code/control/DNRoot.php]: Bit of random whitespace sneaking in? ;-)

**Comment by micmania1** [comment]
Well done, you both passed the test. Cleaned up the clearly intentional whitespace...

**Comment by halkyon** [comment]
Sweet, looks good. I'll wait for tests to pass.

**Pull request merged**